File: app/src/main/java/com/tm/blplayer/utils/ToastUtils.java
package com.tm.blplayer.utils;

import android.content.Context;
import android.widget.Toast;

/**
 * @author wutongming
 * @description
 * @since 2017/5/17
 */

public class ToastUtils {


    public static void showToast(Context context, String text) {
        showShortToast(context, text);
    }

    public static void showToast(Context context, int strId) {
        showShortToast(context, strId);
    }

    public static void showShortToast(Context context, String text) {
        Toast.makeText(context, text, Toast.LENGTH_SHORT).show();
    }

    public static void showShortToast(Context context, int strId) {
        showShortToast(context, context.getResources().getString(strId));
    }

    public static void showLongToast(Context context, String text) {
        Toast.makeText(context, text, Toast.LENGTH_LONG).show();
    }

    public static void showLongToast(Context context, int strId) {
        showShortToast(context, context.getResources().getString(strId));
    }

}

File: app/src/main/java/com/tm/blplayer/utils/ToastUtils.kt
package com.tm.blplayer.utils

import android.content.Context
import android.widget.Toast

/**
 * @author wutongming
 * *
 * @description
 * *
 * @since 2017/5/17
 */

object ToastUtils {


    fun showToast(context: Context, text: String) {
        showShortToast(context, text)
    }

    fun showToast(context: Context, strId: Int) {
        showShortToast(context, strId)
    }

    fun showShortToast(context: Context, text: String?) {
        Toast.makeText(context, text, Toast.LENGTH_SHORT).show()
    }

    fun showShortToast(context: Context, strId: Int) {
        showShortToast(context, context.resources.getString(strId))
    }

    fun showLongToast(context: Context, text: String) {
        Toast.makeText(context, text, Toast.LENGTH_LONG).show()
    }

    fun showLongToast(context: Context, strId: Int) {
        showShortToast(context, context.resources.getString(strId))
    }

}
